### PR TITLE
Remove visible assertions dependency

### PR DESCRIPTION
**Pull request by bsideup**
See https://github.com/testcontainers/testcontainers-java/issues/4008#issuecomment-824617867

### CODE REVIEW

**Comment by martinvw** [comment]
@bsideup thanks a lot!

Please note that it seems that [one build is still failing](https://github.com/testcontainers/testcontainers-java/pull/4010/checks?check_run_id=2417379044)

**Comment by bsideup** [comment]
@martinvw thanks, fixed 👍

File: core/src/main/java/org/testcontainers/containers/GenericContainer.java [hunk 1]
```diff
@@ -748,7 +747,7 @@ private void applyConfiguration(CreateContainerCmd createCommand) {
         boolean shouldCheckFileMountingSupport = binds.size() > 0 && !TestcontainersConfiguration.getInstance().isDisableChecks();
         if (shouldCheckFileMountingSupport) {
             if (!DockerClientFactory.instance().isFileMountingSupported()) {
-                VisibleAssertions.warn(
+                System.err.println(
```
Comment (rnorth) [inline core/src/main/java/org/testcontainers/containers/GenericContainer.java:750]: Should we just make this use SLF4J? I know VisibleAssertions previously emitted to stdout, but it seems that we should probably take this opportunity to just consistently log to SLF4J...

Comment (bsideup) [reply core/src/main/java/org/testcontainers/containers/GenericContainer.java:750]: I guess we could for this one, yes :)

**Review by rnorth — APPROVED** [review]

**Pull request merged**